Makefile

```diff
@@ -0,0 +1,134 @@
+ARCH:=$(shell uname -m)
+
+skogul: $(wildcard *.go */*.go */*/*.go)
+	@echo 🤸 go build !
+	@go build -ldflags "-X main.versionNo=$V" -o skogul ./cmd/skogul
+
+docs/skogul.rst: skogul
+	@echo 😽 Generating documentation $@
+	@./skogul -make-man > $@
+
+skogul.1: docs/skogul.rst
+	@echo 🎢 Generating man-file $@
+	@rst2man < $< > $@
+
+notes: docs/NEWS
+	@echo ⛲ Extracting release notes.
+	@./build/release-notes.sh ${GIT_DESCRIBE} > notes
+
+# MAGIC (I hate noisy Make-runs)
+%/:
+	@mkdir -p $@
+
+all: skogul skogul.1 docs/skogul.rst
+
+install: skogul skogul.1 docs/skogul.rst
+	@echo 🙅 Installing
+	@install -D -m 0755 skogul ${DESTDIR}${PREFIX}/bin/skogul
+	@install -D -m 0644 skogul.1 ${DESTDIR}${PREFIX}/share/man/man1/skogul.1
+	@install -D -m 0644 docs/examples/default.json ${DESTDIR}/etc/skogul/default.json
+	@cd docs; \
+	find -type f -exec install -D -m 0644 {} ${DESTDIR}${DOCDIR}/{} \;
+	@install -D -m 0644 README.rst LICENSE -t ${DESTDIR}${DOCDIR}/
+
+
+# Any complaints on this macro-substitution without patches and I introduce m4.
+rpm-prep/SPECS/skogul.spec: build/redhat-skogul.spec.in | rpm-prep/SPECS/
+	@echo  ❕Building spec-file
+	@cat $< | sed "s/xxVxx/${GIT_DESCRIBE}/g; s/xxARCHxx/${ARCH}/g; s/xxVERSION_NOxx/${VERSION_NO}/g" > $@
+	@which dpkg >/dev/null 2>&1 && { echo 🆒 Adding debian-workaround for rpm build; sed -i 's/^BuildReq/\#Debian hack, auto-commented out: BuildReq/g' $@; } || true
+
+# Build RPM. The spec has a blank %prep, so it assumes sources are already
+# available. This isn't perfect, since it creates a tight coupling between
+# Makefile and specfile, but it isn't all that bad either, since it allows
+# building with minimal redundant effort, and without having to commit to
+# git.
+rpm: rpm-prep/SPECS/skogul.spec | rpm-prep/BUILDROOT/
+	@echo 🎇 Triggering huge-as-heck rpm build
+	@DEFAULT_UNIT_DIR=/usr/lib/systemd/system ;\
+	RPM_UNIT_DIR=$$(rpm --eval $%{_unitdir}) ;\
+	if [ "$${RPM_UNIT_DIR}" = "$%{_unitdir}" ]; then \
+	    echo "😭 _unitdir not set, setting _unitdir to $$DEFAULT_UNIT_DIR"; \
+	    rpmbuild --quiet --bb \
+	        --nodebuginfo \
+	    	--build-in-place \
+		--define "_rpmdir $$(pwd)" \
+		--define "_topdir $$(pwd)" \
+		--define "_unitdir $$DEFAULT_UNIT_DIR" \
+		--buildroot "$$(pwd)/rpm-prep/BUILDROOT" \
+		rpm-prep/SPECS/skogul.spec; \
+	else \
+	    rpmbuild --quiet --bb \
+	        --nodebuginfo \
+	    	--build-in-place \
+		--define "_rpmdir $$(pwd)" \
+		--define "_topdir $$(pwd)" \
+		--buildroot "$$(pwd)/rpm-prep/BUILDROOT" \
+		rpm-prep/SPECS/skogul.spec; \
+	fi
+	@cp x86_64/skogul-${VERSION_NO}-1.x86_64.rpm .
+	@echo ⭐ RPM built: ./skogul-${VERSION_NO}-1.x86_64.rpm
+
+check: test fmtcheck vet
+
+vet:
+	@echo 🔬 Vetting code
+	@go vet ./...
+
+fmtcheck:
+	@echo 🦉 Checking format with gofmt -d -s
+	@if [ "x$$(find -name '*.go' -not -wholename './gen/*' -exec gofmt -d -s {} +)" != "x" ]; then find -name '*.go' -not -wholename './gen/*' -exec gofmt -d -s {} +; exit 1; fi
```

Comment:
```
$ make fmtcheck
🦉 Checking format with gofmt -d -s
find: illegal option -- n
```

By adding the path to search in (".") directly after `find` (`find . -name ...`) everything works. That's required as per my find (BSD version). Note, there are two statements requiring the change. Both inside the if and in the running code if the conditional is true. It's relevant for `fmtfix` too.

Reply:
I think you forgot to add it for fmtfix. Fmtcheck works.